docs/guides/es2k/building-p4cp-acc.md

```diff
@@ -0,0 +1,152 @@
+# Building P4 Control Plane for the ACC
+
+This document explains how to build P4 Control Plane for the ARM Compute
+Complex (ACC) of the Intel&reg; IPU E2100.
+
+## Prepare the System
+
+There are several things to do before you can build P4 Control Plane.
+
+- Install CMake 3.15 or above
+
+  Avoid versions 3.24 and 3.25. They cause the dependencies build to fail.
+
+- Install OpenSSL 1.1
+
+  Note that P4 Control Plane is not compatible with BoringSSL.
+
+- Install the ACC SDK
+
+  See [Installing the ACC SDK](installing-acc-sdk.md) for instructions.
+
+## Install the Stratum Dependencies
+
+See [Building Stratum Dependencies for the ACC](../../../setup/building-es2k-stratum-deps.md)
+for instructions.
+
+You will need the Host dependencies, Target dependencies, and Build environment
+setup file, in order to cross-compile P4 Control Plane for the ACC.
+
+## Fetch the Source Code
+
+If you have not already done so, you will need to get a copy of the IPDK
+networking-recipe repository and its submodules:
+
+```bash
+git clone --recursive https://github.com/ipdk-io/networking-recipe.git ipdk.recipe
+```
+
+You may substitute your own local directory name for `ipdk.recipe`.
+
+## Define the Environment
+
+First, change to the source directory:
+
+```bash
+cd ipdk.recipe
+```
+
+Source the file that defines the
+[target build environment variables](../../../setup/building-es2k-stratum-deps.md#5-defining-the-target-build-environment).
```

Comment:
Suggestion: Instead of doing a relative path, can we add the link to document and section directly. This way our documents are relocatable.

Reply:
What do you mean by "adding the link to the document and section directly?"

Reply:
I meant something like this `https://github.com/ipdk-io/networking-recipe/blob/main/setup/building-es2k-stratum-deps.md#5-defining-the-target-build-environment`

Reply:
Ah: full URL. Relative links make development easier, especially when the documents you're working on haven't been committed. I figured I'd change to URLs as part of moving the Stratum dependencies to their own repository (at which point they will become necessary).

Reply:
That sounds good too. I am thinking of suggesting that we make these documents part of IPU SDK. We can scrub the documents for relative links if and when we do.